deployment/cdk/opensearch-service-migration/cdk.context.json

```diff
@@ -42,6 +42,8 @@
     "// help capture/target proxy pt. 1 of 2": "captureProxyService and targetClusterProxyService require networking access configured to successfully deploy,",
     "// help capture/target proxy pt. 2 of 2": "consider deploying without first and enabling after ensuring cluster networking access on the migration console",
     "captureProxyServiceEnabled": false,
-    "targetClusterProxyServiceEnabled": false
+    "captureProxyDesiredCount": 0,
+    "targetClusterProxyServiceEnabled": false,
+    "targetClusterProxyDesiredCount": 0
```

Comment:
@AndreKurait What do you think about making these ma console commands rather than associating this behavior to deployment?

Reply:
I'm thinking about mirroring the existing behavior for RFS which is deployed with 0 instances by default and then interactively scaled up